samcli/commands/build/command.py

```diff
@@ -63,7 +63,7 @@
 @click.option('--build-dir', '-b',
               default=DEFAULT_BUILD_DIR,
               type=click.Path(file_okay=False, dir_okay=True, writable=True),  # Must be a directory
-              help="Path to a folder where the built artifacts will be stored")
+              help="Path to a folder where the built artifacts will be stored. If a clean build, this directory will be first removed.")
```

Comment:
we always do a clean build. So the 'if' shouldn't be necessary

Reply:
The wording was future looking. I can remove. Do we even mention clean build is all we support anywhere today?